craft_providers/lxd/launcher.py

```diff
@@ -92,6 +92,15 @@ def _create_instance(
             project=project,
         )
 
+    # set the full instance name as image description
+    lxc.config_set(
```

Comment:
Should this be indented (to be part of the if statement above)?  
I think this only needs to be written when the base instance is created.

Reply:
Yes, this is suppose to be one more indent, but somehow I formatted it wrong.